deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: unifiedpush-operator
           # Replace this with the built image name
-          image: quay.io/aerogear/unifiedpush-operator:master
+          image: aliok/upsoprsep26_1145
```

Comment:
You probably committed this by accident or convenience of testing but just flagging it.